scripts/s3mi

```diff
@@ -326,6 +330,34 @@ def adjust_RAM_params():
         MAX_CONCURRENT_REQUESTS = 15
 
 
+def get_credentials():
+    """
+    On EC2/ECS, the AWS CLI calls the instance/container metadata service to fetch instance profile/role credentials
+    every time it runs, unless it finds credentials in environment variables or config files.
+
+    There is a rate limit on the metadata service, and we trigger it sometimes with the following error:
+    Error when retrieving credentials from container-role: Error retrieving metadata: Received non 200 response
+    (429) from ECS metadata: You have reached maximum request limit.
+
+    To avoid this, we fetch the credentials using standard botocore logic and cache them in a process-wide cache.
+    These credentials are short-lived, so we refresh them if we last cached them more than 10 minutes ago.
+    This behavior is only active when the constructor is called with cache_aws_credentials=True.
```

Comment:
I don't think you're using `cache_aws_credentials`, so probably best to remove this line.

Reply:
Thanks